onnxscript/values.py

```diff
@@ -284,7 +284,8 @@ def __init__(
         self._param_schemas: Optional[tuple[ParamSchema, ...]] = None
 
         if self._op_schema is None:
-            logging.debug(
+            logger = logging.getLogger("onnxscript")
```

Comment:
Feel free to move this to the top of the module in the global namespace and use `logger = logging.getLogger(__name__)`. Thanks!

Reply:
Thanks, just did that.